.github/workflows/test.yaml

```
@@ -0,0 +1,47 @@
+# .github/workflows/test.yaml
+
+name: Test
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  smoketest-matrix:
+    name: Smoke Test
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        goos: [linux]
+        goarch: [amd64, arm64]
+    steps:
+    - uses: actions/checkout@v3
+    - uses: actions/setup-go@v5
+    - uses: docker/setup-qemu-action@v3
```

```suggestion
    - uses: docker/setup-qemu-action@v3
```

Why do we need qemu actions? :)

---

To test arm64 binaries on amd64 systems.